test-validator/src/lib.rs

```diff
@@ -719,6 +752,14 @@ impl TestValidator {
         RpcClient::new_with_commitment(self.rpc_url.clone(), CommitmentConfig::processed())
     }
 
+    /// Return a nonblocking RpcClient for the validator.
+    pub fn get_nonblocking_rpc_client(&self) -> nonblocking::rpc_client::RpcClient {
```

Comment:
`get_nonblocking_rpc_client()` is a bit of a long `fn` name as well, but again I have nothing better atm

Reply:
This one seems fine to me.

Reply:
This one really feels like it should be `get_async_rpc_client()`...